docs/edit-contracts.md

```diff
@@ -0,0 +1,235 @@
+
+Open the file `docker/zkevm-contracts.Dockerfile` and make the following edits:
+
+```diff
+diff --git a/docker/zkevm-contracts.Dockerfile b/docker/zkevm-contracts.Dockerfile
+index 1a18e9c..531f667 100644
+--- a/docker/zkevm-contracts.Dockerfile
++++ b/docker/zkevm-contracts.Dockerfile
+@@ -12,10 +12,13 @@ LABEL description="Helper image to deploy zkevm contracts"
+ # STEP 1: Download zkevm contracts dependencies and compile contracts.
+ ARG ZKEVM_CONTRACTS_BRANCH
+ WORKDIR /opt/zkevm-contracts
++
++# TEMPORARY: clone from my local storage instead of github.
++COPY local-test-zkevm-contracts .
++
+ # FIX: `npm install` randomly fails with ECONNRESET and ETIMEDOUT errors by installing npm>=10.5.1.
+ # https://github.com/npm/cli/releases/tag/v10.5.1
+-RUN git clone --branch ${ZKEVM_CONTRACTS_BRANCH} https://github.com/0xPolygonHermez/zkevm-contracts . \
+-  && npm install --global npm@10.6.0 \
++RUN npm install --global npm@10.6.0 \
+   && npm install \
+   && npx hardhat compile
+```
+
+This change tells Docker to build the image `zkevm-contracts` from your new local version of that repo instead of fetching it from github.
+
+Next let's rebuild our edited docker image `zkevm-contracts`. Open a terminal in your repo `kurtosis-cdk`, move to the `docker` directory, and rebuild the image:
+
+```bash
+cd docker
+docker build . \
+ --tag local/zkevm-contracts:fork9 \
+ --build-arg ZKEVM_CONTRACTS_BRANCH=v6.0.0-rc.1-fork.9 \
+ --build-arg POLYCLI_VERSION=main \
+ --file zkevm-contracts.Dockerfile
+```
+
+Your new image should now be visible in docker. The command
+
+```bash
+docker images --filter "reference=local/zkevm-contracts"
+```
+
+should produce output like
+
+```
+REPOSITORY              TAG       IMAGE ID       CREATED          SIZE
+local/zkevm-contracts   fork9     fbd050369e61   22 minutes ago   2.37GB
+```
+
+# Spin up a devnet with your new docker image
+
+Let's spin up a new Polygon CDK devnet with your edited docker image `zkevm-contracts`.
+
+Point your devnet to your local docker image. Open the file `params.yml` and change `leovct/zkevm-contracts` to `local/zkevm-contracts`:
+
+```diff
+diff --git a/params.yml b/params.yml
+index 5293da7..5e451b6 100644
+--- a/params.yml
++++ b/params.yml
+@@ -55,7 +55,7 @@ args:
+   zkevm_da_image: 0xpolygon/cdk-data-availability:0.0.7
+   # zkevm_da_image: 0xpolygon/cdk-data-availability:0.0.6
+
+-  zkevm_contracts_image: leovct/zkevm-contracts # the tag is automatically replaced by the value of /zkevm_rollup_fork_id/
++  zkevm_contracts_image: local/zkevm-contracts # the tag is automatically replaced by the value of /zkevm_rollup_fork_id/
+
+   # zkevm_agglayer_image: 0xpolygon/agglayer:0.1.3
+   zkevm_agglayer_image: ghcr.io/agglayer/agglayer-rs:main
+```
+
+Optional: by default, your devnet disables fancy dashboards. If you want to view your devnet from a fancy dashboard such as Grafana then open the file `params.yml` and ensure that `deploy_observability` is set to `true`.
```

Comment:
nit: we have slightly changed the way we define additional services such as Grafana

For reference: https://github.com/0xPolygon/kurtosis-cdk/blob/main/params.yml#L48

```suggestion
Optional: by default, your devnet disables fancy dashboards. If you want to view your devnet from a fancy dashboard such as Grafana then open the file `params.yml` and ensure that `args.additional_services` includes `"prometheus_grafana"`.